src/bundle/Resources/public/scss/_content-tree.scss

```diff
@@ -17,10 +17,10 @@
         transform-origin: left;
 
         &--animate {
-            transition: all 0.2s ease-out;
+            transition: all 0.3s ease-out;
```

Comment:
I think we should reuse there the variable from this PR https://github.com/ezsystems/ezplatform-admin-ui/pull/1375/files